### DISCUSSION TITLE
Combining raw files for the search in DIA NN

### DISCUSSION THREAD

**Discussion by DianaSamodova** [vdemichev/DiaNN#559] [Q&A] [answered]
Dear Vadim,
I was wondering, whether there is a command line option available for combining several raw MS files into 1 experiment for analysis with DIA NN? I mean, assigning same "experiment name" to multiple raw files (same as in MaxQuant)? Thanks in advance.
Best wishes,
Diana

**Comment by vdemichev** [comment, marked as answer]
Hi Diana,

Yes, you can analyse an arbitrary number of files with DIA-NN. The easiest way to use the command line as opposed to GUI is too copy the parameters the GUI passes to diann.exe (printed in the log window). You can also configure the output file name to whatever is convenient.

Best,
Vadim
